soh/soh/Enhancements/randomizer/3drando/item_location.cpp

```diff
@@ -777,6 +777,43 @@ void LocationTable_Init() {
     locationTable[SONG_FROM_OCARINA_OF_TIME]                     = ItemLocation::Delayed(RC_SONG_FROM_OCARINA_OF_TIME,          0xFF, "Song from Ocarina of Time",                      SONG_FROM_OCARINA_OF_TIME,   SONG_OF_TIME,              {Category::cSong},                               SpoilerCollectionCheck::Chest(0x51, 0x1F), SpoilerCollectionCheckGroup::GROUP_HYRULE_FIELD);
     locationTable[SONG_FROM_WINDMILL]                            = ItemLocation::Delayed(RC_SONG_FROM_WINDMILL,                 0xFF, "Song from Windmill",                             SONG_FROM_WINDMILL,          SONG_OF_STORMS,            {Category::cSong},                               SpoilerCollectionCheck::EventChkInf(0x5B), SpoilerCollectionCheckGroup::GROUP_KAKARIKO);
 
+    /*-------------------------------
+            --- BEEHIVES ---
+      -------------------------------*/
+
+    locationTable[KF_STORMS_GROTTO_BEEHIVE_LEFT]                 = ItemLocation::Base   (RC_KF_STORMS_GROTTO_BEEHIVE_LEFT,          0, "KF Storms Grotto Beehive Left",          KF_STORMS_GROTTO_BEEHIVE_LEFT,          BLUE_RUPEE, {Category::cBeehive}, SpoilerCollectionCheck::RandomizerInf(), SpoilerCollectionCheckGroup::GROUP_KOKIRI_FOREST);
```

Comment:
Each beehive actually has a specific item that it can drop, some have blue rupees and some red rupees, they could also have any dropped item (arrows, bombs, pieces of heart) but the ones in vanilla only have blue and red rupees.